tierkreis/tierkreis/controller/data/models.py

```diff
@@ -22,10 +21,11 @@ class PNamedModel(Protocol):
     """
 
     def _asdict(self) -> dict[str, PType]: ...
-    def __getitem__(self, key: SupportsIndex, /) -> PType: ...
 
+    # def __getitem__(self, key: SupportsIndex, /) -> PType: ...
```

Comment:
I had to comment this out for some reason, but I think I've broken the internal type checking in quite a few places, should look to add it back.